clients/cloud/python/producer.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+#
+# Copyright 2018 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# =============================================================================
+#
+# Produce messages to Confluent Cloud
+# Using Confluent Python Client for Apache Kafka
+#
+# =============================================================================
+
+from confluent_kafka import Producer, KafkaError
+from confluent_kafka.admin import AdminClient, NewTopic
+import json
+import ccloud_lib
+
+
+
+if __name__ == '__main__':
+
+    # Initialization
+    args = ccloud_lib.parse_args()
+    config_file = args.config_file
+    topic = args.topic
+    conf = ccloud_lib.read_ccloud_config(config_file)
+
+    # Create Producer instance
+    p = Producer({
+           'bootstrap.servers': conf['bootstrap.servers'],
+           'sasl.mechanisms': 'PLAIN',
+           'security.protocol': 'SASL_SSL',
+           'sasl.username': conf['sasl.username'],
+           'sasl.password': conf['sasl.password']
+    })
+
+    # Create topic if needed
```

Comment:
I would move this to its own function, and create the topic before the Producer.

Reply:
I disagree, I think AdminClient is a key feature of the python client and topic creation is required for Confluent Cloud.  So it's important to showcase prominently